claim_validate.go

```diff
@@ -0,0 +1,113 @@
+package jwt
+
+import (
+	"errors"
+	"sort"
+	"time"
+)
+
+type JwtPayloadClaimValidator func(tokenClaims *Claims) error
```

Comment:
Don't repeat yourself in the package name [`jwt.JwtPayloadClaimValidator`]. Don't lowercase abbreviations [`Jwt`]. Do `jwt.Validator` perhaps?